jupyter_launcher_shortcuts/__init__.py

```diff
@@ -70,3 +70,9 @@ def _jupyter_nbextension_paths():
         'src': 'static',
         "require": "jupyter_launcher_shortcuts/tree"
     }]
+
+def _jupyter_labextension_paths():
```

Comment:
We should also rename `_jupyter_server_extension_paths` to `_jupyter_server_extension_points` c.f. https://github.com/jupyter-server/jupyter_server/blob/f2d40f692c29e155349b179d57a454e7f23ea898/jupyter_server/extension/utils.py#L80-L93

Reply:
Done!